viewer/src/componentModel.ts

```diff
@@ -254,7 +259,15 @@ export class ComponentModel {
                 "text": text
             });
         }
-        const triggerableLinksJson = modelJson.ComponentData.TransversalLinks[0].TransversalTransitionData || [];
+
+        // let triggerableLinksJson = modelJson.ComponentData.TransversalLinks === undefined ? [] : modelJson.ComponentData.TransversalLinks[0].TransversalTransitionData;
+        let triggerableLinksJson;
```

Comment:
your code seems to be too complicated (why checking undefined twice ?)
ex:
const triggerableLinksJson = modelJson.ComponentData.TransversalLinks && modelJson.ComponentData.TransversalLinks.length > 0 ? odelJson.ComponentData.TransversalLinks[0].TransversalTransitionData : [];

Reply:
done